styles/globals.css

```diff
@@ -4,10 +4,23 @@
 @tailwind variants;
 
 @layer components {
+    .surface-shadow {
+        @apply shadow-[0_1px_3px_rgba(0,0,0,0.12),0_1px_2px_rgba(0,0,0,0.24)];
+    }
+    .surface {
```

Comment:
Please explain your intention here

Reply:
It seems to be a work of the second party working in this repository. Thanks.